packages/frontend-shared/src/locales/en-US.json

```diff
@@ -111,8 +111,8 @@
         "chooseAComponentHeader": "Choose a component"
       },
       "importEmptySpec": {
-        "header": "Create a new spec",
-        "description": "We'll generate an empty spec file to start testing components.",
+        "header": "Create starter spec",
```

Comment:
I think we should keep the header as `Create new spec`  and just update the description to use the updated term of starter spec. I don't have a super strong opinion on starter vs template -- I've just never heard the term starter before which is why I suggested template.

Reply:
Sounds good, I'll make the changes suggested in the closed PR's comments & ping you guys when I'm done :)